fake-project/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.16
```

Comment:
Should this be `1.0.4`?

Reply:
This file doesn't matter; it's overwritten by the script in `Dockerfile`. It's only here to make it work reasonably when testing locally.